pkg/madmin/health.go

```diff
@@ -20,167 +20,650 @@ package madmin
 import (
 	"context"
 	"encoding/json"
-	"io"
+	"errors"
 	"net/http"
 	"net/url"
+	"runtime"
+	"syscall"
 	"time"
 
-	"github.com/minio/minio/pkg/disk"
-	"github.com/minio/minio/pkg/net"
-
-	smart "github.com/minio/minio/pkg/smart"
-	"github.com/shirou/gopsutil/v3/cpu"
-	diskhw "github.com/shirou/gopsutil/v3/disk"
-	"github.com/shirou/gopsutil/v3/host"
-	"github.com/shirou/gopsutil/v3/mem"
-	"github.com/shirou/gopsutil/v3/process"
+	"github.com/shirou/gopsutil/cpu"
+	"github.com/shirou/gopsutil/disk"
+	"github.com/shirou/gopsutil/host"
+	"github.com/shirou/gopsutil/mem"
+	"github.com/shirou/gopsutil/net"
+	"github.com/shirou/gopsutil/process"
 )
 
-// HealthInfo - MinIO cluster's health Info
-type HealthInfo struct {
-	TimeStamp time.Time       `json:"timestamp,omitempty"`
-	Error     string          `json:"error,omitempty"`
-	Perf      PerfInfo        `json:"perf,omitempty"`
-	Minio     MinioHealthInfo `json:"minio,omitempty"`
-	Sys       SysHealthInfo   `json:"sys,omitempty"`
+// HealthInfoVersion is current health info version.
+const HealthInfoVersion = "1"
```

Comment:
```suggestion
const HealthInfoVersion = "2"
```

We can set the version to `2` as the current version is considered as `v1` in `health-analyzer`

Reply:
closing as the code has moved to `madmin-go`